backend/app/Recommendation/Domain/Model/ValueObjects/Query/Query.php

```diff
@@ -6,7 +6,7 @@
 use App\Common\Domain\ValueObject;
 use Exception;
 
-class Query extends ValueObject
+class Query
```

Comment:
Нужно обозначить, что класс принадлежит ValueObject

```suggestion
class Query extends ValueObject
```